### ISSUE TITLE
webapp: some fonts show up more than once in tabs

### ISSUE THREAD

**Issue by felipesanches** [googlefonts/fontbakery#1130]
Among the check results tabs, there are sometimes a few repetitions of font filenames. We should forbid that making the tabs unique (a single one per font file).

![Example](https://cloud.githubusercontent.com/assets/213676/19712629/3d6c5622-9b1e-11e6-847b-240ff071f9cb.png)

**Comment by felipesanches** [comment]
After some testing of the webapp, it seems to me that the repetition described here is actually caused by multiple runs of the tool. (This is probably a duplicate of issue #1134) Even after reloading the webpage I still get some leftover from a previous run. Only after restarting the webserver I get another clean set of check results.

I'm not investigating what exactly in the python code it causing this.

**Comment by felipesanches** [comment]
Yeah, definitely a duplicate of #1134 ! And it makes that higher priority (very high!), as it messes up with more than a single use of the tool.
